<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools</class>
 <widget class="QMainWindow" name="Tools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="txtOutput">
      <property name="font">
       <font>
        <family>Tahoma</family>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/Close-16.png</normaloff>:/icons/Close-16.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExec">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/play-16.png</normaloff>:/icons/play-16.png</iconset>
   </property>
   <property name="text">
    <string>Exec</string>
   </property>
   <property name="toolTip">
    <string>Execute Command</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/edit-clear-16.png</normaloff>:/icons/edit-clear-16.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear Output</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
